client/src/styles/ToggleSwitchStyle.ts

```diff
@@ -23,19 +21,19 @@ export const SwitchSelection = styled.span`
   z-index: 1;
   top: 0px;
   left: 0px;
-  width: 45px;
```

Comment:
love the implementation of rem units that can change with the size of the parent component or screen size